test/features/support/hooks.js

```diff
@@ -0,0 +1,92 @@
+// Before And After hooks used while feature executes
+
+const outputDir = './reports/'
+const screenshotDir = './reports/screenshots/'
+const targetJson = outputDir + 'cucumber_report.json'
+
+const JsonFormatter = require('cucumber').JsonFormatter
+const fse = require('fs-extra')
+const reporter = require('cucumber-html-reporter')
+
+const {defineSupportCode} = require('cucumber')
+
+defineSupportCode(function ({registerHandler}) {
+  registerHandler('BeforeFeatures', function () {
+    const configData = require('../../data/config.json')
+    signIn = require('../page_objects/signInPage')
```

Comment:
all these variables are not declared and also I don't see them being used anywhere.
Is that still work in progress?